openreview/venue_request/process/deployProcess.py

```diff
@@ -5,6 +5,7 @@ def process(client, note, invitation):
     SUPPORT_GROUP = GROUP_PREFIX + '/Support'
     conference = openreview.helpers.get_conference(client, note.forum, SUPPORT_GROUP, setup=True)
     conference.create_submission_stage()
+    conference.invitation_builder.set_venue_template_invitations()
```

Comment:
where does invitation_builder has set_venue_template_invitations method

i saw self.invitation_builder = invitation.InvitationBuilder(client) in builder.py
but in invitation.py there's no set_venue_template_invitations